instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestStreamHandler.java

```diff
@@ -50,11 +51,16 @@ protected TracingRequestStreamHandler(AwsLambdaTracer tracer) {
   }
 
   @Override
-  public final void handleRequest(InputStream input, OutputStream output, Context context)
+  public void handleRequest(InputStream input, OutputStream output, Context context)
       throws IOException {
-    Span span = tracer.startSpan(context, Kind.SERVER);
+
+    InputStreamFactory streamFactory = InputStreamFactory.forStream(input);
```

Comment:
Do you think it's possible to do this only if the propagator is configured?

Reply:
Great idea, will implement a switch,